challenge.rb

```diff
@@ -6,9 +6,13 @@
 # and should return either `true` or `false` based on whether `age` is less than `minimum_age`.
 
 def is_of_age? age, minimum_age
-  # Replace this comment with your code.
+  if age.to_i >= minimum_age.to_i
+    return true
+  else
+    return false
+  end
 end
-
+#tested
```

Comment:
This is correct! However, I want to point out that saying
```
if some_condition
  return true
else 
  return false
end
```
is kind of redundant. Why? If the condition is true, we should return `true`, and if the condition is false, we should return `false`. So why not just return the condition?
```
def is_of_age? age, minimum_age
  age.to_i >= minimum_age.to_i
end
```